data/card_catalogue.json

```diff
@@ -1148,47 +1110,47 @@
         {
           "title": "Cure",
           "upgrade": "Restore",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Heal 8 to character in melee range",
+            "rules": "Heal 8 to a visible character",
             "flavour": "\"Yeah that feels a little better\" -Hugo, recently decapitated",
             "image": "assets/core/card/illustration/Cure.png"
           }
         },
         {
           "title": "Restore",
           "upgrade": "Renew",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Heal 10 to target visible character",
+            "rules": "Heal 10 to a visible character",
             "flavour": "\"Well that sealed up the wound but it isn't really helping\" -Hugo, recently decapitated",
             "image": "assets/core/card/illustration/Restore.png"
           }
         },
         {
           "title": "Renew",
           "upgrade": "Reincarnate",
-          "tier": "green",
+          "tier": "red",
           "details": {
-            "rules": "Heal 30 to a character in melee range",
+            "rules": "Heal 30  to a visible character",
             "flavour": "\"I feel better than ever. Except for my body, which I can no longer feel\" -Hugo, recently decapitated",
             "image": "assets/core/card/illustration/Renew.png"
           }
         },
         {
           "title": "Reincarnate",
           "upgrade": "",
-          "tier": "red",
+          "tier": "gold",
           "details": {
-            "rules": "Revive a dead character. Return this card to the deck",
```

Comment:
Just not needed. If you have a gold card and want to revive one character a session I'm cool with that.